app/sass/app.scss

```diff
@@ -9,4 +9,5 @@
 @import 'atomic.scss';
 @import 'selectOptions.scss';
 @import 'sql.scss';
-@import 'print.scss';
\ No newline at end of file
+@import 'print.scss';
+@import 'brmodelo.scss';
```

Comment:
Actually we don't need. I tried all possibilities and couldn't find a thing where this class have an impact.
I put it because it was in git history a customisations I did on joint.css file.

Anyways, it is deleted now.

thanks for the attentive eyes